tutor/templates/apps/openedx/config/cms.env.yml

```diff
@@ -27,8 +27,6 @@ CELERY_BROKER_USER: "{{ REDIS_USERNAME }}"
 CELERY_BROKER_PASSWORD: "{{ REDIS_PASSWORD }}"
 ALTERNATE_WORKER_QUEUES: "lms"
 ENABLE_COMPREHENSIVE_THEMING: true
-COMPREHENSIVE_THEME_DIRS: ["/openedx/themes"]
-STATIC_ROOT_BASE: "/openedx/staticfiles"
```

Comment:
Note for reviewers: Since [this edx-platform change](https://github.com/openedx/edx-platform/pull/34554), we can specify these as environment variables (in the Dockerfile) and they will be automatically loaded into Django settings.